lib/clients/backbeat-2017-07-01.api.json

```diff
@@ -74,6 +74,68 @@
                 "payload": "Location"
             }
         },
+        "MultipleBackendPutObject": {
+            "http": {
+                "method": "PUT",
+                "requestUri": "/_/backbeat/multipleBackendPutObject/{Bucket}/{Key+}"
```

Comment:
Let's use lowercase for the route `/_/backbeat/multiplebackendput/{Bucket}/{Key+}`. Case-sensitive urls add confusion.

Reply:
Updated.